### PR TITLE
feat: implement boot loading and error screens

### PR DESCRIPTION
**Pull request by satazor**
See: https://github.com/ipfs-shipyard/pm-idm/issues/121

Also:
- Moved lock-screen inside boot
- Moved some logic inside app to boot because we will need it for authenticate & signing "mini-apps"
- Remove unused components

### CODE REVIEW

File: src/boot/activity-detector/ActivityDetector.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { throttle } from 'lodash';
+import { connectIdmWallet } from 'react-idm-wallet';
+
+const THROTTLE_TIME = 10000;
```
Comment (PedroMiguelSS) [inline src/boot/activity-detector/ActivityDetector.js:6]: I think we can increase this number to something close to our locker minimum accepted time. This way, the restart will happen less often.

**Review by PedroMiguelSS — APPROVED** [review]
👍

File: src/boot/activity-detector/ActivityDetector.js (hunk 1 above)
Comment (satazor) [reply src/boot/activity-detector/ActivityDetector.js:6]: Yep we can but this way we do not need to read the maxTime nor to react to changes to it.

**Review by dominguesgm — APPROVED** [review]
Looks good to me 👍

**Pull request merged**